source/geod24/LocalRest.d

```diff
@@ -87,6 +86,8 @@ private struct Response
     /// `true` if the method returned successfully,
     /// `false` if an `Exception` occured
     bool success;
+    /// `true` if the request timed-out
+    bool timed_out;
```

Comment:
I'd prefer making `success` an `enum` at this point

Reply:
Good point.